### DISCUSSION TITLE
Can't Install any app on CasaOS?

### DISCUSSION THREAD

**Discussion by Shastarage** [IceWhaleTech/CasaOS#1795] [Q&A] [answered]
Hi! i've just installed CasaOS on my linux server, tried to install Apps from the appstore and i get the exact same Error from everything. 

![Screenshot_7](https://github.com/IceWhaleTech/CasaOS/assets/41430424/f31685bd-4ec0-4742-92aa-0d74a210e643)

> Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /DATA/AppDaa/uptimekuma/app/data

I get the same for Plex, Jellyfin and everything lol.

The only thing i did was install a fresh version of Linux, install CasaOS, then run. Nothing else. 

thanks in advance to any help :D

**Comment by ZixX3r** [comment]
What version of CasaOs are you on? afaik 0.48 has an issue installing apps so 0.47 is recommended for the time being.

**Reply by Shastarage** [reply, marked as answer]
The problem was that Ubuntu server downloaded docker during installation, then CasaOS installed docker on top of that somehow, causing a major bug.
i had to uninstall both CasaOS and Docker completely, then install CasaOS again and i got it working. :D
